Functions/Administration/xASL_adm_DefineASLSequence.m

```diff
@@ -1,44 +1,44 @@
-function [x] = xASL_adm_DefineASLSequence(x, bVerbose)
+function [xQ] = xASL_adm_DefineASLSequence(xQ, bVerbose)
 %xASL_adm_DefineASLSequence Obtain ASL sequence type for sequence-specific
 %image processing
 %
-% FORMAT: [x] = xASL_adm_DefineASLSequence(x, bVerbose)
+% FORMAT: [xQ] = xASL_adm_DefineASLSequence(xQ, bVerbose)
 %
 % INPUT:
-%   x                - x structure containing all input parameters (REQUIRED)
-%   x.Q.readoutDim   - dimensionality of readout (2D or 3D) (OPTIONAL)
-%   x.Q.Vendor       - Either 'GE', 'Philips', 'Siemens' (OPTIONAL)
+%   xQ                - x.Q structure containing all input parameters (REQUIRED)
+%   xQ.readoutDim   - dimensionality of readout (2D or 3D) (OPTIONAL)
+%   xQ.Vendor       - Either 'GE', 'Philips', 'Siemens' (OPTIONAL)
 %   bVerbose         - verbose output (OPTIONAL, DEFAULT=true)
 %
 % OUTPUT:
-%   x               - x structure containing all output parameters
-%   x.Q.Sequence    - sequence type (readout)
+%   xQ               - x structure containing all output parameters
+%   xQ.Sequence    - sequence type (readout)
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % DESCRIPTION: This ExploreASL function tries to check what ASL sequence is
-% being processed, if this was not already defined in x.Q.Sequence.
+% being processed, if this was not already defined in xQ.Sequence.
 % It does so by checking known combinations of readout dimensionality
-% (x.Q.readoutDim) and Vendor, knowing the product sequences of the Vendors.
+% (xQ.readoutDim) and Vendor, knowing the product sequences of the Vendors.
 %
-% EXAMPLE: x = xASL_adm_DefineASLSequence(x);
+% EXAMPLE: xQ = xASL_adm_DefineASLSequence(xQ);
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
-% Copyright 2015-2023 ExploreASL
+% Copyright 2015-2024 ExploreASL
 
 
-%% Check quantification fields readoutDim & x.Q.Sequence
+%% Check quantification fields readoutDim & xQ.Sequence
 if nargin<2 || isempty(bVerbose)
     bVerbose = true;
 end
 
-if ~isfield(x.Q, 'readoutDim') || isempty(x.Q.readoutDim)
-    warning('x.Q.readoutDim parameter missing');
+if ~isfield(xQ, 'readoutDim') || isempty(xQ.readoutDim)
+    warning('xQ.readoutDim parameter missing');
 end
 
 % Check for illegal sequence definitions
-if isfield(x.Q,'Sequence')
-    if isempty(regexpi(x.Q.Sequence, '(2d_epi|3d_grase|3d_spiral)'))
-        warning(['Unknown ASL sequence: ' xASL_num2str(x.Q.Sequence)]);
+if isfield(xQ,'Sequence')
+    if isempty(regexpi(xQ.Sequence, '(2d_epi|3d_grase|3d_spiral)'))
+        warning(['Unknown ASL sequence: ' xASL_num2str(xQ.Sequence)]);
```

Comment:
True. And it can become annoying NEvertheless. If x.Q.Sequence is provided, it should be correct. And it shouldn't really happen that we don't have vendor+readoutDim.